x-pack/plugins/session_view/public/components/process_tree/index.tsx

```diff
@@ -145,14 +145,16 @@ export const ProcessTree = ({
       if (processEl) {
         processEl.prepend(selectionAreaEl);
 
-        const cTop = scrollerRef.current.scrollTop;
-        const cBottom = cTop + scrollerRef.current.clientHeight;
+        const { height: elHeight, y: elTop } = processEl.getBoundingClientRect();
+        const { y: viewPortElTop, height: viewPortElHeight } =
+          scrollerRef.current.getBoundingClientRect();
 
-        const eTop = processEl.offsetTop;
```

Comment:
There was a bug in this check where isVisible would return true even if the process was below the fold. The improved code was borrowed from the isVisible hook.